helm/pattern-2/apim-gw-km-with-analytics/templates/wso2apim-is-as-km-deployment.yaml

```diff
@@ -28,78 +28,63 @@ spec:
   template:
     metadata:
       labels:
-        deployment: wso2apim-km
+        deployment: wso2apim-is-as-km
     spec:
       containers:
-      - name: wso2apim-km
-        image: docker.wso2.com/wso2am:2.5.0
+      - name: wso2apim-is-as-km
+        image: docker.wso2.com/wso2is-km:5.7.0-ubuntu
```

Comment:
@ThilinaManamgoda `5.7.0-ubuntu` is not a valid image tag for WSO2 Docker images.